### DISCUSSION TITLE
Superadmin key and passowrd in latest docker image

### DISCUSSION THREAD

**Discussion by grindsa** [Keyfactor/ejbca-ce#224] [Q&A] [answered]
Hi,

I am installing ejbca according to the [Quick Start Guide - Start EJBCA Container with Client Certificate Authenticated Access](https://doc.primekey.com/ejbca/tutorials-and-guides/quick-start-guide-start-ejbca-container-with-client-certificate-authenticated-access)

I am using your latest [docker image](https://hub.docker.com/layers/primekey/ejbca-ce/latest/images/sha256-b6bc33884ba22c5ad6b373bfcb258e8f26b8d8097a33abecf5e91302944129ce?context=explore). 

In your latest image the banner has been changed and i am not able to get superadmin key and password (as described in step 2.2 of your guide) anymore.

````
 *****************************************************************************************
 * A fresh installation was detected and a ManagementCA was created for your initial     *
 * access to the system.                                                                 *
 *                                                                                       *
 *   URL:      [https://ejbca:443/ejbca/adminweb/](https://ejbca/ejbca/adminweb/)
 *                                                                                       *
 * If you use different port mapping or a proxy, please adapt the URL above accordingly. *
 *****************************************************************************************

````

How can i get superadmin cert and key now? Or is there any chance to create these credentials by script? Any advice or hint to further documentation.

Thank you for your help and have a nice day..

/G

**Comment by primetomas** [comment, marked as answer]
You are right, thanks for the report. Defaults changed but doc was not updated properly. We are working on it.
You can add "-e TLS_SETUP_ENABLED=true" to start with the classic workflow.

**Comment by grindsa** [comment]
Thank you very much. With the "-e TLS_SETUP_ENABLED=true" everything works as expected...
